### PR TITLE
Palette Helper Changes

### PR DESCRIPTION
**Pull request by SuicSoft**
When changing colors with palette helper they fade quickly (duration can be changed) to make it smooth looking.

### CODE REVIEW

**Comment by SuicSoft** [comment]
Fixed all compile errors

**Comment by ButchersBoy** [comment]
...yes...let's not make anything static.  If anything we should introduce an IPaletteHelper interface to help people with DI, TDD etc

File: MaterialDesignThemes.Wpf/PaletteHelper.cs [hunk 1]
```diff
@@ -123,23 +123,31 @@ public void ReplaceAccentColor(string name)
...
                 return true;
             }
 
             foreach (var dictionary in parentDictionary.MergedDictionaries)
-            {
-                if (ReplaceEntry(entryName, newValue, dictionary))
-                    return true;
-            }
-
+                return ReplaceEntry(entryName, newValue, dictionary);
```
Comment (Davipb) [inline MaterialDesignThemes.Wpf/PaletteHelper.cs]: The 'if' there was necessary: In your new code, the ReplacyEntry method now returns false if the first merged dictionary fails to have the entry, never looking at the others.

**Comment by SuicSoft** [comment]
Fixed more errors (the if thingy).
Don't know if the iscolor is needed. If all the new values are colors you don't need it. So removed it

**Comment by SuicSoft** [comment]
There should be no problems with the code now. The change is that when a color is changed with PaletteHelper it will fade to the new color. The fade can be customized

**Pull request closed without merging**